source/iface/queue.rst

```diff
@@ -227,6 +227,10 @@ Wait for all enqueued tasks to complete.
 Wait for all enqueued tasks and pass asynchronous errors to handler
 provided in :ref:`queue-constructors`.
 
+.. rubric:: Example
+
+See :ref:`exception-handling-example <exception-handling-example>`.
```

Comment:
I would have expected you to make the title 'Exception Handling Example' instead of 'Example 1' and then this reference would be: :ref:`exception-handling-example` so it would say:
See Exception Handling Example
and Exception .... would be a link. I will merge now and we can discuss the best way to title/refer to examples afterwards.